src/client/game.cpp

```diff
@@ -1875,14 +1858,14 @@ void Game::processUserInput(f32 dtime)
 
 void Game::processKeyInput()
 {
-	if (wasKeyDown(KeyType::DROP)) {
-		dropSelectedItem(isKeyDown(KeyType::SNEAK));
-	} else if (wasKeyDown(KeyType::AUTOFORWARD)) {
+	if (input->wasKeyDown(KeyType::DROP)) {
```

Comment:
why?
The exact point of this is to avoid the need to type `input->` repeatedly and you're adding this back.

Reply:
As far as I can tell, the point of these functions wasn't less typing. They originally had different functionality from their `input->` counterparts, which was then later removed in a cleanup commit (that incidentally broke joystick support in the process): 99c9e7a9864a8da5aaf97e615d2bdefa1295497a

I think typing `input->` each time is better than having additional code that does *nothing* but call another function.

Reply:
That explains the (old) other "DO NOT use isKeyDown" comment, which now makes no sense as the function is equivalent.
I still think it's useful to keep it for less typing and so this PR doesn't have to do 100s of line changes.

Reply:
> so this PR doesn't have to do 100s of line changes.

That's why the "cleanup" part is its own commit. I personally prefer having the `input->` prefix, but I'm fine either way. The only important thing to me is that it's consistent across the file, which will (IIRC) require the 100 lines of code change either way.

Reply:
removed the cleanup commit (the one adding `input->`)